screens/visualization/sketch.py

```diff
@@ -165,82 +181,68 @@ def __call__(self):
         x = screen_x + a * np.cos(f*y + p)
         screen_pos += [x, y]
 
-    for figure in figures:

```

Comment:
So, I'd keep this iteration and the whole `if/elif/...` structure inside `__main__` but now call the function instead of having the code in-line.